### PR TITLE
Allow 's around values in parse_key_value / parse_logfmt

### PR DESCRIPTION
**Pull request by maxstarkov**
This pull request for #125.

### CODE REVIEW

**Review by pront — APPROVED** [review]
This LGTM but let's also wait for @fuchsnj's approval.

**Review by fuchsnj — COMMENTED** [review]
The changes look good to me. Please just add a line to `changelog.md` with a link to this PR.

**Review by jszwedko — APPROVED** [review]
Thanks for adding the changelog entry @maxstarkov !

**Pull request merged**
